application/frontend/controllers/MfaController.php

```diff
@@ -152,7 +161,7 @@ public function actionList(string $employeeId): array
      * @throws BadRequestHttpException
      * @throws NotFoundHttpException
      */
-    protected function getRequestedMfa($id)
+    protected function getAuthorizedMfa($id)
```

Comment:
What does "authorized" mean here? Is this just meant to return verified MFA records? Or something else? (Thanks)

Reply:
It also checks that the user is authorized to deal with that Mfa.
Let me know if you prefer a different name for it.

Reply:
Revert to `getRequestedMfa($id)` (per discussion)